### PR TITLE
Copy-edit axes_grid tutorial.

### PR DESCRIPTION
**Pull request by anntzer**
Probably still needs further work (not in this PR), but hopefully an
improvement over the current state...

## PR Checklist

- [ ] Has pytest style unit tests (and `pytest` passes).
- [ ] Is [Flake 8](https://flake8.pycqa.org/en/latest/) compliant (run `flake8` on changed files to check).
- [ ] New features are documented, with examples if plot related.
- [ ] Documentation is sphinx and numpydoc compliant (the docs should [build](https://matplotlib.org/devel/documenting_mpl.html#building-the-docs) without error).
- [ ] Conforms to Matplotlib style conventions (install `flake8-docstrings` and run `flake8 --docstring-convention=all`).
- [ ] New features have an entry in `doc/users/next_whats_new/` (follow instructions in README.rst there).
- [ ] API changes documented in `doc/api/next_api_changes/` (follow instructions in README.rst there).

### CODE REVIEW

File: tutorials/toolkits/axes_grid.py [hunk 1]
```diff
@@ -206,98 +165,77 @@
...
    :scale: 50
 
-   Simple Anchored Artists
-
-
 InsetLocator
 ------------
 
-:mod:`mpl_toolkits.axes_grid1.inset_locator` provides helper classes
+:mod:`.axes_grid1.inset_locator` provides helper classes
```
Comment (jklymak) [inline tutorials/toolkits/axes_grid.py:199]:
```suggestion

.. seealso::

    `.Axes.inset_axes` and `.Axes.indicate_inset_zoom` in the main library.

:mod:`.axes_grid1.inset_locator` provides helper classes
```

File: tutorials/toolkits/axes_grid.py [hunk 2]
```diff
@@ -206,98 +165,77 @@
...
 
     inset_axes = zoomed_inset_axes(ax,
-                                   0.5, # zoom = 0.5
+                                   0.5,  # zoom = 0.5
                                    loc='upper right')
 
-creates an inset axes whose data scale is half of the parent axes.
-Here is complete examples.
+creates an inset axes whose data scale is half of the parent axes.  This can be
+useful to show a zoomed-up area of the parent axes:
```
Comment (jklymak) [inline tutorials/toolkits/axes_grid.py:222]:
```suggestion
useful to show the zoomed area on the parent axes:
```

**Review by jklymak — COMMENTED** [review]
Couple of small suggestions.  I think throughout we should try and point to main-library equivalents when they exist.

**Comment by anntzer** [comment]
Thanks.  Agree with referring to the main lib where possible.

**Review by QuLogic — APPROVED** [review]

File: tutorials/toolkits/axes_grid.py [hunk 3]
```diff
@@ -1,115 +1,84 @@
...
 
-A grid of Axes.
-
-In Matplotlib, the axes location (and size) is specified in normalized
-figure coordinates. This may not be ideal for images that needs to be
-displayed with a given aspect ratio; for example, it is difficult to
-display multiple images of a same size with some fixed padding between
-them.  `~.axes_grid1.axes_grid.ImageGrid` can be used in such a case; see its
-docs for a detailed list of the parameters it accepts.
+In Matplotlib, axes location and size are normally specified in normalized
```
Comment (QuLogic) [inline tutorials/toolkits/axes_grid.py:30]: Drop doubled 'normal'
```suggestion
In Matplotlib, axes location and size are usualy specified in normalized
```

File: tutorials/toolkits/axes_grid.py [hunk 4]
```diff
@@ -1,115 +1,84 @@
...
+figure coordinates (0 = bottom left, 1 = top right), which makes
+it difficult to achieve a fixed (absolute) padding between images.
+`~.axes_grid1.axes_grid.ImageGrid` can be used to achieve such a padding; see
+its docs its detailed API.
```
Comment (QuLogic) [inline tutorials/toolkits/axes_grid.py:34]:
```suggestion
its docs for detailed API information.
```

File: tutorials/toolkits/axes_grid.py [hunk 5]
```diff
@@ -1,115 +1,84 @@
...
 
-* axes in the same column has a same axes width (in figure
-  coordinate), and similarly, axes in the same row has a same
-  height. The widths (height) of the axes in the same row (column) are
-  scaled according to their view limits (xlim or ylim).
+* Axes in a same column share their x-axis, and axes in a same row share their
+  y-axis (in the sense of `~.Axes.sharex`, `~.Axes.sharey`).  Additionally,
+  Axes in a same column all have the same width, and axes in a same row all
+  have the same height.  These widths and heights are scaled in proportion to
+  the axes' view limits (xlim or ylim).
```
Comment (QuLogic) [inline tutorials/toolkits/axes_grid.py:51]: The 'the' to 'a' change doesn't read right; either `Axes in a column` or back to `Axes in the same column`.

File: tutorials/toolkits/axes_grid.py [hunk 6]
```diff
@@ -206,98 +165,80 @@
...
    :align: center
    :scale: 50
 
-   Inset Locator Demo2
-
-
-RGB Axes
-~~~~~~~~
+RGBAxes
+~~~~~~~
```
Comment (QuLogic) [inline tutorials/toolkits/axes_grid.py:241]: Wrong level
```suggestion
RGBAxes
-------
```

**Comment by anntzer** [comment]
thanks, edited.

**Pull request merged**